conf/kong-entrypoint.sh

```diff
@@ -11,4 +11,10 @@ else
     done
 fi
 
-exec "$@"
\ No newline at end of file
+# remove hosts configs for busted tests
+sed -i '/dns_hostsfile = spec\/fixtures\/hosts/d' /kong/spec/kong_tests.conf
```

Comment:
What happens if we keep it ?

Reply:
```
[==========] Running tests from scanned files.
[----------] Global test environment setup.
[----------] Running tests from /kong-plugin/my-plugin/spec/my-plugin/access_spec.lua
./spec/helpers.lua:95: [postgres error] could not retrieve current migrations: [postgres error] [toip() name lookup failed]: dns server error: 3 name error

stack traceback:
	./spec/helpers.lua:95: in function 'run_migrations'
	...-trust-auth/spec/my-plugin/access_spec.lua:34: in function <...-trust-auth/spec/my-plugin/access_spec.lua:33>
```

Reply:
Ah right, it breaks the Docker hosts

Reply:
By removing the set hosts config for busted tests, It does not come up with the same error anymore.

Reply:
Yea, looks like the only impact this will have is on uses of `lambda.us-east-1.amazonaws.com`.